tests/infra/server.py

```diff
@@ -44,7 +44,7 @@ def run(self):
         project_dir = os.path.join(os.path.dirname(__file__), "..", "..", "chalice")
         factory = CLIFactory(project_dir=project_dir)
         self._chalice_app = factory.load_chalice_app()
-        self._chalice_app._override_exptime_seconds = 86400  # something large.  sys.maxsize causes chalice to flip.
+        self._chalice_app._override_exptime_seconds = 30  # something large.  sys.maxsize causes chalice to flip.
```

Comment:
Is this needed. I find having this value large makes debugging easier. Otherwise you only have 30 seconds to step though the server code before a timeout error is triggered.

Reply:
If it's 86400, then the timeout code for PUT /bundles with a missing file is not really exercised (i.e., it immediately times out).

Reply:
I suppose when debugging we can override this value with something larger. Will you add a note in data_store.tests.readme that this value can be adjusted when debugging.